index.html

```diff
@@ -0,0 +1,245 @@
+                </div>
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class="d-none d-md-block see-whole">
+                    SEE THE WHOLE PROGRAM
+                </div> 
+                <div class="d-block d-md-none join-summit">
+                    Join CC Global Summit 2023
+                </div> 
+            </div>
+        </div>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="speakers">
+        <div class="text-center">
+            <h2 class="speakers">Featured Speakers</h2>
+        </div>
+        <div class="container-lg pb-3">
+            <div class="row align-items-center justify-content-center pt-5" id="mainSection">
+
+              <!-- speakers dynamic code  -->
+
+            </div>
+            <div class="col-md-4 more-btn text-center">
+                MORE <i class="bi bi-chevron-down"></i>
+            </div>
+        </div>
+      
+    </section>
+
+    <!-- partners  -->
+    <section id="partners">
+        <div class="container-fluid py-5 pt-3">
+            <div class="text-center">
+                <h2 class="partners">Partner</h2>
+                <div class="line"></div>
+            </div>
+
+                <div class="col-md-10 offset-md-1 pb-3 pt-5">
+                    <div class="partners-images">
+                        <div class="text-center">
+                            <img src="assets/partner/mozilla 1.png" alt="mozilla" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/googlekorea 1.png" alt="Google" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/naver 1.png" alt="Never" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/daumkakaologo 1.png" alt="daumkakaologo" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/airbnb 1.png" alt="daumkakaologo" class="img-fluid">
+                        </div>
+                    </div>
+                </div> 
+        </div>
+    </section>
+
+    <!-- footer  -->
+
+    <section id="footer">
+        <div class="container-fluid py-5">
+                <div class="col-md-8 offset-md-2">
+                    <div class="footer-p-logo">
+                        <div class="footer-logo">
+                            <img src="assets/images/logo/techlife.png" alt="mozilla" class="img-fluid logo">
+                        </div>
+                        <p class="footer-p">2023 Creative Commons Korea. Some Rights Reserved.</p>
+                    </div>
+                </div> 
+        </div>
+    </section>
```

Comment:
- [ ] For best semantics tag, I think it would be better if you change this conatiner`<section<>` tag to `<footer>` tag as the container of the footer has to be a `footer` tag. I suggest to apply it in both pages(Home && About)